### PR TITLE
Booking flow visual cleanup

### PR DESCRIPTION
**Pull request by heyitsmaimai**

### CODE REVIEW

File: src/users/cd-create-account.vue [hunk 1]
```diff
@@ -208,42 +211,46 @@
...
-      display: block;
-      font-size: 16px;
-      font-weight: bold;
+      margin-bottom: 5px;
+      display: inline-block;
+      font-size: 14px;
+      font-weight: 700;
+    }
+    &__email {
+      max-width: 110%;
```
Comment (Wardormeur) [inline src/users/cd-create-account.vue]: Uh, what ?

File: src/users/cd-create-account.vue [hunk 2]
```diff
@@ -258,22 +265,32 @@
       margin-top: 4px;
       font-weight: 300;
     }
-    &__dob-picker {
-      padding-left: 0;
-      max-width: 600px;
+    &__dob-picker-wrapper {
+      max-width: 50%;
```
Comment (Wardormeur) [inline src/users/cd-create-account.vue]: Max-width:50% is terrible on mobile

File: src/users/cd-create-account.vue (hunk 1 above)
Comment (heyitsmaimai) [reply src/users/cd-create-account.vue]: oh woop sorry i was playing around with that. will fix

File: src/users/cd-create-account.vue (hunk 2 above)
Comment (heyitsmaimai) [reply src/users/cd-create-account.vue]: why? as in it stays too big or becomes too small?

Comment (heyitsmaimai) [reply src/users/cd-create-account.vue]: changed it for 100% on mobile

**Pull request merged**